### PR TITLE
[ntsc-1.2] Match z_std_dma.c

### PR DESCRIPTION
**Pull request by cadmic**
Thanks to @AngheloAlf and @inspectredc for discussions on how to match the unused strings here

### CODE REVIEW

File: src/code/code_n64dd_800AD410.c [hunk 1]
```diff
@@ -12,6 +12,7 @@ s8 func_801C7740_unknown(void);
 u8 B_80121AE0;
 s8 B_80121AE1;
 u8 B_80121AE2;
+vu8 B_80121AE4;
```
Comment (Dragorn421) [inline src/code/code_n64dd_800AD410.c:15]: shouldn't there be a (unused so far) `B_80121AE3` for this added bss variable to be xxx4

Comment (cadmic) [reply src/code/code_n64dd_800AD410.c]: Good point. Apparently it's used in the n64dd segment as a `vu8`
https://github.com/Decompollaborate/n64dd/blob/6e138715c991bffb26216370c7fa8ebefbb065f9/oot/ne2/undefined_syms_ne2.txt#L55
https://github.com/Decompollaborate/n64dd/blob/6e138715c991bffb26216370c7fa8ebefbb065f9/include/code_functions.h#L33

**Review by Dragorn421 — APPROVED** [review]

**Review by fig02 — APPROVED** [review]

**Comment by fig02** [comment]
needs conflict resolved

**Comment by cadmic** [comment]
thanks

**Pull request merged**